soc/riscv/riscv-privilege/telink_w91/Kconfig.defconfig.series

```diff
@@ -11,7 +11,7 @@ config SOC_SERIES
 
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	int
-	default 10000000
+	default 26660000
```

Comment:
Is this MTIMER clock value described somewhere in datasheet? The value seems starnge...